tests/unit/pages/error/InternalServerError.spec.js

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react'
+import InternalServerError from '~/pages/error/InternalServerError'
+
+describe('InternalServerError page test', () => {
+  beforeEach(() => {
+    render(<InternalServerError />)
+  })
+  it('Test title', () => {

```

Comment:
test description should start from 'should...'

Reply:
+